recipes/sts/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "sts" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  skip: True  # [osx or win]
+  number: 0
+
+source:
+  git_url: https://github.com/OnlinePhylo/sts.git
+  git_rev: v{{ version }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - git
+  host:
+    - cmake
+    - pkg-config
```

Comment:
pkg-config and cmake should be in the build section ... git (if you still needed it) should be in the host section